common/src/main/java/io/homeassistant/companion/android/common/sensors/NfcSensorManager.kt

```diff
@@ -0,0 +1,51 @@
+package io.homeassistant.companion.android.common.sensors
+
+import android.content.Context
+import android.nfc.NfcAdapter
+import io.homeassistant.companion.android.common.R as commonR
+
+class NfcSensorManager : SensorManager {
+    companion object {
+        private const val TAG = "NfcSensor"
+
+        val nfcStateSensor = SensorManager.BasicSensor(
+            "nfc_state",
+            "binary_sensor",
+            commonR.string.sensor_name_nfc_sensor,
+            commonR.string.sensor_description_nfc_sensor,
+            "mdi:nfc",
+            entityCategory = SensorManager.ENTITY_CATEGORY_DIAGNOSTIC,
+            updateType = SensorManager.BasicSensor.UpdateType.INTENT
+        )
+    }
+
+    override fun docsLink() = "https://companion.home-assistant.io/docs/core/sensors#nfc-sensor"
+    override val name = commonR.string.sensor_name_nfc_sensor
```

Comment:
Maybe don't name the sensor manager the same as the individual sensor? Usually the pattern is "[NFC] sensor(s)" for the manager and "[NFC] state" for the individual sensor. There are a few that deviate from it but they all are "[thing] sensor" instead of "[thing] state".

(I think "state" is also superfluous but only NFC isn't very descriptive and I don't have a good alternative)

Reply:
Done! Please check my changes if they reflect what you meant :slightly_smiling_face: